Sources/Cloud/Commands/Database/DatabaseFeedbackFormat.swift

```diff
@@ -0,0 +1,25 @@
+public final class DatabaseFeedbackFormat {
+    
+    init(_ console: ConsoleProtocol, _ info: FeedbackInfo) {
+        guard info.status != "exit" else {
+            do {
+                exit(0)
+            } catch {
+                exit(1)
+            }
+        }
+        
+        switch(info.status) {
+        case "error":
+            console.error("[ Error ] " + info.message)
```

Comment:
hmm, why is this a different format than the other errors? can we use the loading bar here?

Reply:
Loading bars will be implemented, but requires a lot of additional work to make it work properly :) .. Again, this is a very early beta, to get the database part of things tested, that will take months to get tested properly, so just want to get it out, so we can start shooting down bugs in the actual databases, sugar can always be added later :)